docs/blog/newsletter-october-2024.rst

```diff
@@ -0,0 +1,134 @@
+* The type of work requested: Charge less for minor editing. Rewriting or creating new content costs more. Re-organizing content (information architecture) or, in a docs-as-code environment, setting up and working with a tool chain would command higher rates.
+* Your skills and experience: Highly skilled and experienced professionals can ask for higher rates.
+
+To calculate an hourly rate (in the US) determine the salary for an equivalent position, divide that by 52x40, and then double or triple that to compensate for benefits. Check job postings and the  `2023 WTD Salary Survey <https://www.writethedocs.org/surveys/salary-survey/2023/>`_ for salaries. Freelancers and consultants can ask for higher rates than contractors.
+
+**Can I assume that the client pays taxes?**
+
+If you’re working as a W2 contractor in the US, the agency takes out federal taxes. As a freelancer (1099), you pay all taxes. In other countries, check your local tax requirements, which vary widely.
+
+**How often will I get paid?**
+
+As a contractor, the pay period depends on the agency. As a freelancer, your contract defines the pay period. Some freelancers bill based on the project (milestones); some calculate invoices based on an hourly rate. 
+
+**What can I consider billable hours?**
+
+As a contractor working on site, you normally bill for hours on site based on your contract. As a freelancer working off site, you bill whenever you work on the project. In general, you don't get overtime unless it’s in your contract.
+
+---------------------------------------
+Using callouts/admonitions in tech docs
+---------------------------------------
+
+When it comes to creating clear and effective technical documentation, knowing when to use callouts and admonitions (boxes that stand out from the surrounding text) and avoiding overuse is key to creating user-friendly content. They can highlight important information and ensure your documentation remains accessible and easy to navigate.
+
+Some style guides follow a tiered system to ensure proper use:
+
+- Tip: Helpful advice that can make tasks easier but isn’t critical.
+- Note: Relevant but non-essential information that adds context but doesn’t affect the task’s outcome.
+- Important: Information that, if missed, could lead to frustration or additional work.
+- Warning: Critical details about data loss or security risks, where overlooking the warning could cause significant problems.
+
+The key is moderation. Too many boxed notes, tips, or warnings can cause “admonition fatigue” and lead to readers skipping over important information. When deciding between boxed content or inline text, weigh the impact. If it’s crucial and might be missed, box it. Otherwise, integrating it into the main body ensures smoother flow and readability.
```

Comment:
```suggestion
The key is moderation. Too many individual notes, tips, or warnings can cause “admonition fatigue” and lead to readers skipping over important information. When deciding between separated content or inline text, weigh the impact. If it’s crucial and might be missed, make it stand out. Otherwise, integrating it into the main body ensures smoother flow and readability.
```